### DISCUSSION TITLE
Please upload to Microsoft Store [SOLVED]

### DISCUSSION THREAD

**Discussion by AlphaCraft9658** [bridge-core/bridge.#322] [Ideas]
Please upload bridge to the Microsoft Store if it's possible. There are some issues with installing it on my Laptop because it says that it may be not safe. Please do this if it is possible and as soon as possible.

**Comment by solvedDev** [comment]
We do not plan to publish bridge. to the Microsoft store. We are currently working on the next major version of the app which will fix the false Windows Defender alarm.
For now, rest assured that the app is completely safe to install. You can literally read every single line of code it executes here on GitHub!

**Reply by AlphaCraft9658** [reply]
Okay. Thank you!
